docs/configuration/index.md

```diff
@@ -24,27 +24,94 @@ The configuration settings are read from two sources:
 
 - the exported [environment variables](#environment-variables) that overrides the configuration specified in the files.
 
-## Updating the configuration at runtime
+## Helm install options
+
+Advanced Helm install options are documented [here](helm-chart-parameters).
+
+## Updating the configuration
 
-You can update the configuration and use `helm upgrade` to update configuration values for the BotKube. Also, you can use `@BotKube` commands which persist the configuration. See the [Usage](../usage/index.md) section for more details.
+To update BotKube configuration, you can either:
 
-You can also change configuration directly in ConfigMap and Secret - is not recommended but is great for quick experimentation.
+- upgrade BotKube installation with Helm,
+- or use dedicated `@BotKube` commands, to e.g. toggle notifications or edit Source Bindings. See the [Usage](../usage/index.md) section for more details.
```

Comment:
you use term `section` so IMO it will be good to link to a given section not whole document 👍 or re-name it to `document`.